applications/nrf5340_audio/src/bluetooth/bt_stream/le_audio.h

```diff
@@ -44,16 +44,10 @@
 /**
  * @brief Callback for receiving Bluetooth LE Audio data.
  *
- * @param	data		Pointer to received data.
- * @param	size		Size of received data.
- * @param	bad_frame	Indicating if the frame is a bad frame or not.
- * @param	sdu_ref		ISO timestamp.
+ * @param	audio_frame	Pointer to audio data struct.
  * @param	channel_index	Audio channel index.
- * @param	desired_size	The expected data size.
  */
-typedef void (*le_audio_receive_cb)(const uint8_t *const data, size_t size, bool bad_frame,
-				    uint32_t sdu_ref, enum audio_channel channel_index,
-				    size_t desired_size);
+typedef void (*le_audio_receive_cb)(struct audio_data *audio_frame, uint8_t channel_index);
```

Comment:
The channel index is already baked into the audio frame.

Reply:
It's not the same. What is in the audio frame is the channel_location. In the channel location, left==1, in the channel_index left==0.